pom.xml

```diff
@@ -43,7 +43,7 @@
     <slf4j.version>1.7.36</slf4j.version>
     <mariadb.version>3.0.5</mariadb.version>
     <hibernate.version>6.1.0.Final</hibernate.version>
-    <tomcat.version>10.1.0-M16</tomcat.version>
+    <tomcat.version>10.1.1</tomcat.version>
```

Comment:
Any reason for upgrade ?

Reply:
Tomcat 10.1.0-* was flagged for vulnerabilities in BDBA